src/step/mod.rs

```diff
@@ -41,6 +41,13 @@ pub(crate) enum Step {
         /// If provided, only issues with this label will be included
         labels: Option<Vec<String>>,
     },
+    /// Search for Gitea issues by status and display the list of them in the terminal.
+    /// User is allowed to select one issue which will then change the workflow's state to
+    /// [`State::IssueSelected`].
```

Comment:
```suggestion
    /// [`Issue::Selected`].
```

This is probably also wrong, I gotta look around the code base and figure out how to explain this better 😅